offer-manager-domain/src/main/groovy/life/qbic/business/products/create/CreateProductInput.groovy

```diff
@@ -0,0 +1,25 @@
+package life.qbic.business.products.create
+
+import life.qbic.datamodel.dtos.business.services.Product
+
+/**
+ * Input interface for the {@link CreateProduct} use case
+ *
+ * @since: 1.0.0
+ *
+ */
+interface CreateProductInput {
+
+    /**
+     * A product is created in the database
+     * @param product The product that is added to the database
+     */
+    void create(Product product)
+
+    /**
+     * Even though a duplicate product in the database exist a new product should be added.
+     * The duplicate product is required to have a different product id!
```

Comment:
```suggestion
     * Even though a duplicate product in the database exists, a new product should be added.
     * The duplicate product is required to have a different product id!
     *
```

Reply:
Will the old product id be ignored and a new product Id provided by the use case?

Reply:
Yes, it is important that we can distinguish between the duplicated packages. Since it is business logic I think it's ok to create the Id in the use case.

Reply:
It would be great if you could state that in the method description shortly since it indicates the expected behaviour of implementing methods. If you want to leave this detail open to the implementation then I would suggest removing `The duplicate product is required to have a different product id!`